app/components/ui/files/publish-modal/component.js

```diff
@@ -17,18 +17,41 @@ export default Ember.Component.extend({
     // An array that holds a pair, (fileName, fileSize)
     fileList: [],
     // Holds an array of objects that the user cannot be exclude from their package
-    nonOptionalFile: ['tale.yaml', 'environment.zip'],
+    nonOptionalFile: ['tale.yaml', 'environment.tar', 'license.txt'],
```

Comment:
I assume we'll be archiving a tar.gz and not just a tar. Also, `license.txt` is now LICENSE.